samples/nrf9160/http_application_update/src/main.c

```diff
@@ -111,12 +112,12 @@ static int led_app_version(void)
 		return 1;
 	}
 
-	gpio_pin_configure(dev, DT_ALIAS_LED0_GPIOS_PIN, GPIO_DIR_OUT);
-	gpio_pin_write(dev, DT_ALIAS_LED0_GPIOS_PIN, 1);
+	gpio_pin_configure(dev, DT_ALIAS_LED0_GPIOS_PIN, GPIO_OUTPUT);
+	gpio_pin_set_raw(dev, DT_ALIAS_LED0_GPIOS_PIN, 1);
```

Comment:
You should rather use the following:
```
	gpio_pin_configure(dev, DT_ALIAS_LED0_GPIOS_PIN,
			   GPIO_OUTPUT_ACTIVE | DT_ALIAS_LED0_GPIOS_FLAGS);
```
and remove the call to `gpio_pin_write()`. Same applies to LED1 below.

Reply:
will do.